docs/supported_apis/dataframe_supported.rst

```diff
@@ -132,7 +145,9 @@ default to pandas.
 | ``ffill``                  | `ffill`_                  | Y                      |                                                    |
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``fillna``                 | `fillna`_                 | P                      | ``value`` parameter of type DataFrame defaults to  |
-|                            |                           |                        | pandas                                             |
+|                            |                           |                        | pandas. **Omnisci**: ``P``, params ``limit``,      |
+|                            |                           |                        | ``downcast`` and ``method`` unsupported. Also      |
+|                            |                           |                        | only ``axis = 0`` supported for now                |
```

Comment:
Why should we mention `axis = 0` case? Isn't it a relational algebra limitation?

Reply:
discussed offline, decided to leave as is.